steps/cadence-genus-synthesis/scripts/designer-interface.tcl

```diff
@@ -57,9 +57,7 @@ set vars(adk_dir) inputs/adk
 set vars(libs_typical,timing) \
     [join "
         $vars(adk_dir)/stdcells.lib
-        [lsort [glob -nocomplain $vars(adk_dir)/stdcells-lvt.lib]]
-        [lsort [glob -nocomplain $vars(adk_dir)/stdcells-ulvt.lib]]
-        [lsort [glob -nocomplain $vars(adk_dir)/stdcells-pm.lib]]
+        [lsort [glob -nocomplain $vars(adk_dir)/stdcells*.lib]]
```

Comment:
This will result in stdcells.lib being in the list twice. You could make the wildcard stdcells-*.lib to prevent this, or remove the first stdcells.lib.

Reply:
The second option is more consistent with what you've done in the DC node.

Reply:
fixed.